backend/lib/endpoints/auth.js

```diff
@@ -124,6 +124,16 @@ async function routes(app) {
       reply.setAuthCookies(token);
       const auth0User = await Auth0.getUser(token);
       const { email_verified: emailVerified } = auth0User;
+      // Ideally, this error should be thrown to comply
```

Comment:
Thanks @emanuelquerty . The code is great - tested it out.

The comment we want but maybe a little more concise? (2 lines max)

```
// Don't throw as already checking 403 from Auth0 (wrong email/pw)
// to respond to client as 401 unauthorized error
```

Reply:
Thanks for the feedback. I will edit the comments and make it more concise (2 lines max)